### DISCUSSION TITLE
Query on Multiple Lake Packages in a Single Model

### DISCUSSION THREAD

**Discussion by YozeGholami** [modflowpy/flopy#2337] [General]
Hi all,

I attempted to incorporate multiple Lake Packages within a single model, but I encountered an issue. Although I can see three Lake Packages in my NAM file of model, only the last defined package is recognized as part of the model. This results in an error. Is it possible to successfully integrate multiple Lake Packages in one model? How can I resolve this error?

Thank you for your assistance.

Best regards,
Amin
![Multiple lake packages](https://github.com/user-attachments/assets/680d2896-4f66-4d6d-be04-6d8a16d81ad0)

**Comment by martclanor** [comment]
Have you tried specifying a unique `filename` and `pname` for each `Modflowgwflak` object?
